hikaricp/metadata.csv

```diff
@@ -0,0 +1,15 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
+hikaricp.connections,gauge,,,,Number of connection,0,hikaricp,connection number,
+hikaricp.connections.timeout.count,count,,,,Total number of timeout connections,0,hikaricp,connection timeout total,
+hikaricp.connections.active,gauge,,,,Number of active connections,0,hikaricp,active connection number,
+hikaricp.connections.idle,gauge,,,,Number of idle connections,0,hikaricp,idle connection number,
+hikaricp.connections.pending,gauge,,,,Number of pending connections,0,hikaricp,pending connection number,
+hikaricp.connections.acquire.seconds.count,count,,,,Count of acquire connection time.,0,hikaricp,acquire connections time seconds,
+hikaricp.connections.acquire.seconds.max,gauge,,,, Max of acquire connection time.,0,hikaricp,acquire connections time seconds,
+hikaricp.connections.acquire.seconds.sum,count,,,,Sum of acquire connection time.,0,hikaricp,acquire connections time seconds,
+hikaricp.connections.creation.seconds.count,count,,,,Count of creation connection time.,0,hikaricp,creation connections time seconds,
+hikaricp.connections.creation.seconds.max,gauge,,,,Max of creation connection time.,0,hikaricp,creation connections time seconds,
+hikaricp.connections.creation.seconds.sum,count,,,,Sum of creation connection time.,0,hikaricp,creation connections time seconds,
+hikaricp.connections.usage.seconds.count,count,,,,Count of usage connection time.,0,hikaricp,usage connections time seconds,
+hikaricp.connections.usage.seconds.max,gauge,,,,Max of usage connection time.,0,hikaricp,usage connections time seconds,
+hikaricp.connections.usage.seconds.sum,count,,,,Sum of usage connection time.,0,hikaricp,usage connections time seconds,
```

Comment:
```suggestion
hikaricp.connections.acquire.seconds.count,count,,,,Count of acquire connection time in seconds.,0,hikaricp,acquire connections time seconds,
hikaricp.connections.acquire.seconds.max,gauge,,,, Max of acquire connection time in seconds.,0,hikaricp,acquire connections time seconds,
hikaricp.connections.acquire.seconds.sum,count,,,,Sum of acquire connection time in seconds.,0,hikaricp,acquire connections time seconds,
hikaricp.connections.creation.seconds.count,count,,,,Count of creation connection time in seconds.,0,hikaricp,creation connections time seconds,
hikaricp.connections.creation.seconds.max,gauge,,,,Max of creation connection time in seconds.,0,hikaricp,creation connections time seconds,
hikaricp.connections.creation.seconds.sum,count,,,,Sum of creation connection time in seconds.,0,hikaricp,creation connections time seconds,
hikaricp.connections.usage.seconds.count,count,,,,Count of usage connection time in seconds.,0,hikaricp,usage connections time seconds,
hikaricp.connections.usage.seconds.max,gauge,,,,Max of usage connection time in seconds.,0,hikaricp,usage connections time seconds,
hikaricp.connections.usage.seconds.sum,count,,,,Sum of usage connection time in seconds.,0,hikaricp,usage connections time seconds,
```

Reply:
Suggesting this for added clarity since the `unit_name` column is not specified here.

Reply:
I would suggest to use the `unit_name` so it will be clearly displayed in the metrics explorer

Reply:
Thanks, i will use `unit_name` then